js/src/lib/interfaces/Libraries.ts

```diff
@@ -521,6 +534,12 @@ export const MODEL_LIBRARIES_UI_ELEMENTS: Partial<Record<ModelLibraryKey, Librar
 		repoUrl:  "https://github.com/fastai/fastai",
 		snippet:  fastai,
 	},
+	"mindspore": {
```

Comment:
My understanding is that mindspore is the name of the framework, and tinyms is the name of the library.

Is all MindSpore model using tinyms, if so we can mix the two. Or do you have other libraries on top of MindSpore?

Reply:
@xianbaoqian Thank you for your review. Yes, you are right, the mindspore is the name of the framework, and tinyms is the name of the library. All the MindSpore models can use tinyms, MindSopore's easy-to-use suite library, to adapt to HF to provide more AI experience services for users.
 I don't know what the repoUrl field in the Libraries.ts file is for, so I fill in tinyms to help the background download directly to tinyms to execute the task. If this field has no special technical purpose, It can be filled with the MindSpore framework name, I have updated, thank you.

Reply:
I see. Thanks. I think it looks good now. 

Curious how do tinyms tells which the task of the model? Thx

Reply:
TinyMS determines what tasks need to be done based on the network and the data set. For example, LeNet networks and mnist datasets perform the image-classification task. I don't know if I've answered your question.